apps/nextjs-website/src/components/MUIIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { HelpOutlineOutlined } from '@mui/icons-material';
+import { AllowedMUIIcon } from '@/lib/fetch/types/mui/AllowedIcon';
+
+// Return requested MUI Icon
+// This solution was preferred over the previous usage of <Icon>
+// With this solution, we can avoid importing the Material Icons font
+
+const iconMapping = {
+  // Button Icons
+  HelpOutlineOutlined: <HelpOutlineOutlined />,
+};
+
+const MUIIcon = (name: AllowedMUIIcon): JSX.Element => iconMapping[name];
```

Comment:
I would define the entry keys based on `iconMapping`, this way we can remove `AllowedMUIICon`

```suggestion
const MUIIcon = (name: keyof typeof iconMapping): JSX.Element =>
  iconMapping[name];
```